src/compas_eve/ghpython/components/Ce_MqttTransport/code.py

```diff
@@ -0,0 +1,40 @@
+"""
+Connect or disconnect to an MQTT broker.
+
+COMPAS EVE v0.5.0
+"""
+
+from threading import Event
+
+from compas_ghpython import create_id
+from ghpythonlib.componentbase import executingcomponent as component
+from scriptcontext import sticky as st
+
+from compas_eve.mqtt import MqttTransport
+
+
+class MqttTransportConnect(component):
```

Comment:
perhaps `ConnectComponent` is more consistent with the other names you used...

Reply:
yep, done at https://github.com/compas-dev/compas_eve/pull/15/commits/ec62b7712cacc1daa28ecb5e44c630472e3cb7da